recipe/build.sh

```diff
@@ -11,7 +27,26 @@ FINAL_DEBUG_FFLAGS=FINAL_DEBUG_FFLAGS_${ctng_target_platform_u}
 FINAL_LDFLAGS=FINAL_LDFLAGS_${ctng_target_platform_u}
 FINAL_CONDA_PYTHON_SYSCONFIGDATA_NAME=FINAL_CONDA_PYTHON_SYSCONFIGDATA_NAME_${ctng_target_platform_u}
 
-if [ -z "${!FINAL_CFLAGS}" ]; then
+FINAL_CFLAGS="${!FINAL_CFLAGS}"
+FINAL_CXXFLAGS="${!FINAL_CXXFLAGS}"
+FINAL_FFLAGS="${!FINAL_FFLAGS}"
+FINAL_DEBUG_CFLAGS="${!FINAL_DEBUG_CFLAGS}"
+FINAL_DEBUG_CXXFLAGS="${!FINAL_DEBUG_CXXFLAGS}"
+FINAL_DEBUG_FFLAGS="${!FINAL_DEBUG_FFLAGS}"
+FINAL_LDFLAGS="${!FINAL_LDFLAGS}"
+FINAL_CONDA_PYTHON_SYSCONFIGDATA_NAME="${!FINAL_CONDA_PYTHON_SYSCONFIGDATA_NAME}"
```

Comment:
I tried this code locally and it is making these strings empty. Is that what was intended?

Reply:
ack ignore this.